packages/std/src/errors/std_error.rs

```diff
@@ -535,6 +535,15 @@ pub enum CheckedMultiplyRatioError {
     Overflow,
 }
 
+#[derive(Error, Debug, PartialEq, Eq)]
+pub enum CheckedFromRatioError {
+    #[error("Denominator must not be zero")]
+    DivideByZero,
+
+    #[error("Multiplication overflow")]
+    MulOverflow,
```

Comment:
Yeah, I knew you'd point it out. In previous PR you mention to not double `multiplication` since it was in name of enum anyway. Now that was not the case, though this whole extra enum still feels redundant for me.